bindings_generator/src/lib.rs

```diff
@@ -167,7 +167,8 @@ fn generate_class_bindings(
 
     // methods and method table
     {
-        generate_method_table(output_method_table, class)?;
+        let has_underscore = api.api_underscore.iter().any(|name| class.name.starts_with(name));
```

Comment:
If `api_underscore` is changed into a `HashSet<String>` then you can use `.contains(&class.name)` here instead.

All classes that need this underscore are all part of the set, so you can test for equality rather than `.starts_with` which should be a lot cheaper :D

Reply:
Are you sure? Nothing quite like a good O(n*m) linear search to wake the CPU up in the morning.